README.md

```diff
@@ -52,9 +51,8 @@ When deploying the multi cluster traffic controller using the make targets the f
     ```
 
 ## 2. Running the controller locally:
-1. Create two env files:
-    * One called `aws-credentials.env` containing **AWS_ACCESS_KEY_ID** and **AWS_SECRET_ACCESS_KEY**
-    * One called `controller-config.env` containing **AWS_DNS_PUBLIC_ZONE_ID** and **ZONE_ROOT_DOMAIN**
+1. Create env files:
+    * One called `aws-credentials.env` containing **AWS_ACCESS_KEY_ID**, **AWS_SECRET_ACCESS_KEY** and **AWS_REGION**
```

Comment:
Is the controller-config.env file needed anymore?
If not, we can remove it from the steps below 
```
    (export $(cat ./controller-config.env | xargs) && export $(cat ./aws-credentials.env | xargs) && make build-controller install run-controller)
```

Reply:
I just added this back. It's not actually need by the controller, but i have added a config to generate a ManagedZone as part of local-setup using the contents of that file see : 

```
./bin/kustomize --reorder none --load-restrictor LoadRestrictionsNone build config/local-setup/controller | kubectl apply -f -
```

Reply:
Nice